doc/api/fs.md

```diff
@@ -3528,6 +3528,13 @@ On Linux, positional writes don't work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+On Windows, if the file descriptor is connected to the console (e.g. `fd == 1`
+or `stdout`) a string encoded as UTF-8 or Unicode (`ucs2`) will not be rendered
+properly.
+It is possible to configure the console to render UTF-8 properly by changing the
+active codepage with the `chcp 65001` command. See the [chcp][] docs for more
+details.
```

Comment:
The two sentences contradict each other. "won't be rendered properly" and "will be rendered properly if you configure it".

Reply:
Yes, probably missing a "by default".